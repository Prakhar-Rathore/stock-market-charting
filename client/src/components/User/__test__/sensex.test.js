import React from "react";
import ReactDOM from "react-dom/client";
import Sensex from "./../sensex";
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { act } from 'react-dom/test-utils';

it("renders without crashing", async() => {
  const root = ReactDOM.createRoot(document.createElement("div"));
  await act( async () => render(<Sensex/>));
});

it("renders correctly", () => {
  render(<Sensex/>)
})

it("matches snapshot", () => {
  const tree = renderer.create(<Sensex/>).toJSON();
  expect(tree).toMatchSnapshot();
})