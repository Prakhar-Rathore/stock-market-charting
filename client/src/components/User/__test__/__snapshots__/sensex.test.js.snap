// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    className="MuiGrid-root css-1hp6q6g-MuiGrid-root"
  >
    <button
      className="MuiButton-root MuiButton-white MuiButton-whitePrimary MuiButton-sizeMedium MuiButton-whiteSizeMedium MuiButtonBase-root  css-1w1rijm-MuiButtonBase-root-MuiButton-root"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "background": "white",
          "color": "black",
          "margin": "5px",
          "marginBottom": "0",
        }
      }
      tabIndex={0}
      type="button"
    >
      3D
    </button>
    <button
      className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "background": "white",
          "color": "black",
          "margin": "5px",
          "marginBottom": "0",
        }
      }
      tabIndex={0}
      type="button"
    >
      1W
    </button>
    <button
      className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "background": "white",
          "color": "black",
          "margin": "5px",
          "marginBottom": "0",
        }
      }
      tabIndex={0}
      type="button"
    >
      1M
    </button>
          
  </div>
</div>
`;
